src/main/java/uk/ac/ebi/subs/validator/biosamples/BiosamplesValidator.java

```diff
@@ -76,38 +75,16 @@ private SingleValidationResult validateSampleName(Sample sample) {
      * @param sample
      */
     private SingleValidationResult validateReleaseDate(Sample sample) {
-        List<Attribute> attributes = sample.getAttributes();
         SingleValidationResult singleValidationResult = generateDefaultSingleValidationResult(sample.getId());
 
-        // FIXME - This is a temporary solution, it will be changed once the release date is an actual field and not an attribute.
-        List<Attribute> releaseDates = attributes.stream()
-                .filter(
-                attribute -> attribute.getName().toLowerCase().equals("release"))
-                .collect(Collectors.toList());
+        LocalDate releaseDate = sample.getReleaseDate();
 
-        if (releaseDates.size() > 1) {
-            singleValidationResult.setValidationStatus(SingleValidationResultStatus.Error);
-            singleValidationResult.setMessage(MULTIPLE_DATES_ERROR);
-            return singleValidationResult;
-        }
-
-        if (releaseDates.isEmpty() || (releaseDates.get(0) != null && (releaseDates.get(0).getValue() == null || releaseDates.get(0).getValue().isEmpty()))) {
+        if (releaseDate == null) {
             singleValidationResult.setValidationStatus(SingleValidationResultStatus.Error);
             singleValidationResult.setMessage(MISSING_DATE_VALUE);
             return singleValidationResult;
         }
 
-
-        String releaseDate = releaseDates.get(0).getValue();
-        try {
-            DateTimeFormatter formatter = DateTimeFormatter.ISO_DATE_TIME;
-            LocalDateTime.parse(releaseDate, formatter);
-        } catch (Exception e) {
-            logger.debug("Invalid date format: " + releaseDate);
-            singleValidationResult.setValidationStatus(SingleValidationResultStatus.Error);
-            singleValidationResult.setMessage(DATE_WRONG_FORMAT + ": " + e.getMessage());
```

Comment:
This has already been approved, but this section on validation of the `releaseDate` format shouldn't have been deleted. BioSamples accepts a specific date format that we need to validate before accepting it.